.gitignore

```diff
@@ -3,3 +3,4 @@ public
 resources
 .DS_Store
 .idea
+.vscode
```

Comment:
I would normally agree with adding an exception for `.vscode` but in our case some relevant config that is used by Gitpod resides in that folder.

Reply:
Removed